server/summary_biplot_server.R

```diff
@@ -1,3 +1,295 @@
-output$SUMMARY_BIPLOT = renderPlot({
-    summary_biplot(1, 1)
+
```

Comment:
im thinking about efficiency, once we have a lot of tabs in this app. maybe instead of multiple table, we could have multiple apps that all live inside the same domain (which we'd manage either on a server that we have full access to, or maybe multiple apps can interact with each other within shinyapps pro version). you'll be in gee land for a while, so no need to get into this, but next time one of us is focusing on app development, let's see what our new turbo shinyapps account allows.